calblueprint/siren
# questions structured

# ✨ New in this PR ✨

## Structured question components 🚨 

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [ ] Explored different packages to use for the question components
 - [ ] Implemented and styled questions components
 - [ ] Pull from Firestore to render questions
 - [ ] Connect submitting answers to Firestore
 - [ ] Complete Intake Form layout

## How can the reviewer test your code? 👩‍💻
run 'expo start' and head to the forms page

The layout should look like the following:

![image](https://user-images.githubusercontent.com/57604304/139561305-bc04044e-2aa2-407c-b5aa-1f1c58d7eeb7.png)

## Any bugs you encountered or still having trouble with? 🐛
The current bugs I am running into are moving the calendar setting to extend the entire width and pulling from the Firestore. The next steps I will be looking into will be pulling the questions from the Firestore and mapping each document to a specific question component.


## Resources 📔
https://blog.logrocket.com/complete-guide-textinput-react-native/
https://hossein-zare.github.io/react-native-dropdown-picker-website/docs/usage/

🧜‍♀️ cc: @shannonbonet @phoebeli23
